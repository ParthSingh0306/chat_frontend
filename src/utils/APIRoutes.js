export const host = "https://chat-backend-hb9o.onrender.com";
export const loginRoute = `${host}/api/v1/users/login`;
export const registerRoute = `${host}/api/v1/users/register`;
export const logoutRoute = `${host}/api/v1/users/logout`;
export const allUsersRoute = `${host}/api/v1/users/allusers`;
export const setAvatarRoute = `${host}/api/v1/users/setavatar`;
export const sendMessageRoute = `${host}/api/v1/messages/addmsg`;
export const recieveMessageRoute = `${host}/api/v1/messages/getmsg`;
