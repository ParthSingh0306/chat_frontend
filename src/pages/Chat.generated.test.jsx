import renderer from 'react-test-renderer';
import React, { useEffect, useState, useRef } from "react";
import { useNavigate } from "react-router-dom";
import axios from "axios";
import { io } from "socket.io-client";
import { allUsersRoute, host } from "../utils/APIRoutes";
import Contacts from "../components/Contacts";
import Welcome from "../components/Welcome";
import ChatContainer from "../components/ChatContainer";
import Chat from "./Chat";

jest.mock("react-router-dom");
jest.mock("axios");
jest.mock("socket.io-client");
jest.mock("../utils/APIRoutes");
jest.mock("../components/Contacts");
jest.mock("../components/Welcome");
jest.mock("../components/ChatContainer");

const renderTree = tree => renderer.create(tree);
describe('<Chat>', () => {
  it('should render component', () => {
    expect(renderTree(<Chat 
    />).toJSON()).toMatchSnapshot();
  });
  
});