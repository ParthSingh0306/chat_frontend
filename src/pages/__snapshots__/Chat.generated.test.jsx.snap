// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chat> should render component 1`] = `
<div
  className="sc-beySPh fwoYfh"
>
  <div
    className="container"
  />
</div>
`;
